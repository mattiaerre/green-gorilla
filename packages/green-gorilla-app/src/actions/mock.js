const mock = {
  info: {
    name: 'red-rhinoceros',
    version: '2.0.0-alpha.4'
  }
};

export default mock;
