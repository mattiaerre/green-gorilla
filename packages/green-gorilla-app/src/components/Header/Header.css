.Header {
  background-color: #222;
  padding: 2em;
  color: white;
}

.Header h1 {
    color: palegreen;
}