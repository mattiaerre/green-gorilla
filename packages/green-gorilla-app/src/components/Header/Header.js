import React from 'react';
import './Header.css';

const Header = ({ name, version }) => {
  return (
    <div className="Header">
      <h1>{name}</h1>
      <h2>{version}</h2>
    </div>
  );
};

export default Header;
