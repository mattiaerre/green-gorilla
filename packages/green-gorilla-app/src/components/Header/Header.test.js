import React from 'react';
import { create } from 'react-test-renderer';
import Header from './Header';
import mock from '../../actions/mock';

test('<Header /> to match snapshot', () => {
  const component = create(
    <Header {...mock.info} />
  );
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});
