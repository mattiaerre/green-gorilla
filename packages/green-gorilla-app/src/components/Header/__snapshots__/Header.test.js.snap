// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> to match snapshot 1`] = `
<div
  className="Header"
>
  <h1>
    red-rhinoceros
  </h1>
  <h2>
    2.0.0-alpha.4
  </h2>
</div>
`;
